src/Protocol/Protocol_1_9.cpp

```diff
@@ -2665,8 +2665,11 @@ void cProtocol_1_9_0::HandlePacketUseEntity(cByteBuffer & a_ByteBuffer)
 	{
 		case 0:
 		{
-			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand)
-			m_Client->HandleUseEntity(EntityID, false);
+			HANDLE_READ(a_ByteBuffer, ReadVarInt, UInt32, Hand);
+			if (Hand == 0)  // TODO: implement handle off-hand (1) actions
```

Comment:
The enum matches the protocol (I added it for that purpose and use `eMainHand` the same way in client settings).  Protocol-wise, the game is serializing an enum by its index (that's what "VarInt Enum" on the wiki indicates), not converting an internal representation to a number, so I think it's fine to also use an enum in cuberite.